### ISSUE TITLE
terminus collapse input directories

### ISSUE THREAD

**Issue by JennyFrodoBrynnNick** [COMBINE-lab/Terminus#12]
sorry for my two earlier postings (terminus error, salmon options), as these were resolved in previous issues. So terminus group is working for me now, but now I have a problem with terminus collapse. I am unsure about the input directories (-d ...) to terminus collapse. Are the input directories to terminus collapse equal to the output directories generated by running terminus group separately on several samples?
The input directories for terminus group are the output directories generated by Salmon (salmon_quant), but I assume that these are not the input directories to terminus collapse (since it needs information from terminus group).
It seems that terminus collapse needs both information from Salmon and from Terminus group, but I do not understand how to accomplish this with the input directories.
So in short, what are the input directories to Terminus collapse?
Thanks, Ina

**Comment by NPSDC** [comment]
Hi Ina,
Apologies for taking so long. Getting back to your question, input directories would be the path to the salmon quantified files or as you mention the output directories generated by Salmon (same as the one given to individual group step). The output for `collapse` should be same as the output of `group`.

More details here
https://terminus.readthedocs.io/en/latest/

Further, if the earlier posts are no longer an issue could you close them.

**Comment by JennyFrodoBrynnNick** [comment]
Thank you. I closed the earlier issues.
Best, Ina



Ina Hoeschele, Ph.D.
Professor, Fralin Life Sciences Institute at Virginia Tech
Professor, Department of Statistics, Virginia Tech
Adjunct Professor, Wake Forest School of Medicine, Winston-Salem NC
Office Phone: (540)231-3135
Cell Phone: (540)239-7264
***@***.***
